src/Truncation/FramesStrategy.php

```diff
@@ -0,0 +1,32 @@
+<?php namespace Rollbar\Truncation;
+
+class FramesStrategy extends AbstractStrategy
+{
+    
+    const FRAMES_OPTIMIZATION_RANGE = 150;
```

Comment:
This probably comes from matching what the Gem does, but I feel like it is really large. Maybe we can make this 75 instead so that we get 150 total frames rather than 300 when we truncate.